### PR TITLE
Add flag to specify docker install PATH

### PR DESCRIPTION
**Pull request by dontmint**
I submit this PR to fix issue #26, please review my change and merge if possible. Thank you for your time.

### CODE REVIEW

**Comment by edeNFed** [comment]
Hi @dontmint,
First, thanks for contributing to kubectl-flame!
Second, I think that in order for this to work you need to propagate this value to the agent and use it in
https://github.com/VerizonMedia/kubectl-flame/blob/43e1acb7bfc7c7e1656f6fa56221658dc56ee34c/agent/utils/filesystem.go#L9-L10

Please let me know if you need any help.

File: cli/cmd/data/target.go [hunk 1]
```diff
@@ -18,6 +18,7 @@ type TargetDetails struct {
 	Alpine        bool
 	DryRun        bool
 	Image         string
+  DockerPath    string
```
Comment (edeNFed) [inline cli/cmd/data/target.go:21]: please add spaces to match the other variables' indentation

Comment (dontmint) [reply cli/cmd/data/target.go]: I added spaces to match indentation.

**Comment by dontmint** [comment]
> Hi @dontmint,
> First, thanks for contributing to kubectl-flame!
> Second, I think that in order for this to work you need to propagate this value to the agent and use it in
> https://github.com/VerizonMedia/kubectl-flame/blob/43e1acb7bfc7c7e1656f6fa56221658dc56ee34c/agent/utils/filesystem.go#L9-L10
> 
> Please let me know if you need any help.

Hi @edeNFed ,
Thanks for letting me know, I'm working on it, can wait to see it works.
Cheers.

**Comment by dontmint** [comment]
> > Hi @dontmint,
> > First, thanks for contributing to kubectl-flame!
> > Second, I think that in order for this to work you need to propagate this value to the agent and use it in
> > https://github.com/VerizonMedia/kubectl-flame/blob/43e1acb7bfc7c7e1656f6fa56221658dc56ee34c/agent/utils/filesystem.go#L9-L10
> > 
> > Please let me know if you need any help.
> 
> Hi @edeNFed ,
> Thanks for letting me know, I'm working on it, can wait to see it works.
> Cheers.

Hello @edeNFed ,

After compiled the `cli` and tested with my local cluster, I think we don't need to change 
https://github.com/VerizonMedia/kubectl-flame/blob/43e1acb7bfc7c7e1656f6fa56221658dc56ee34c/agent/utils/filesystem.go#L9-L10
Because the `agent` process runs inside the container and the PATH inside will not change since I just updated HostPath, the MountPath is still `/var/lib/docker`
https://github.com/dontmint/kubectl-flame/blob/5528e23de9f95da65fb08856503760c19f7830ad/cli/cmd/kubernetes/job/jvm.go#L76

**Review by edeNFed — APPROVED** [review]

**Comment by edeNFed** [comment]
@dontmint Got it, sounds good. Going to merge.

**Pull request merged**